web/beacon-app/src/features/projects/components/ProjectList.tsx

```diff
@@ -43,6 +44,11 @@ function ProjectList() {
         </HelpTooltip>
       </div>
       <div className="flex justify-between rounded-lg bg-[#F7F9FB] px-3 py-2">
+        <div className="mt-2">
+          <button disabled={isFetchingProjects} onClick={getProjects}>
```

Comment:
In this video, you should be able to see where fetching highlights in the React Query dev tools when the refresh button is clicked. https://www.awesomescreenshot.com/video/17097918?key=f104eee28cf7ed39d50996057432c49c